PurchasesTests/LocalReceiptParsing/Builders/InAppPurchaseBuilderTests.swift

```diff
@@ -244,7 +251,7 @@ class InAppPurchaseBuilderTests: XCTestCase {
 private extension InAppPurchaseBuilderTests {
 
     func sampleInAppPurchaseContainerWithMinimalAttributes() -> ASN1Container {
-        return containerFactory.inAppPurchaseContainerFromContainers(containers: minimalAttributes())
```

Comment:
Does this produce an error? There's no mention of it in the `SwiftStyleGuide`, and IMO it's a matter of preference.

Reply:
Oops, I misunderstood [this](https://github.com/RevenueCat/purchases-ios/blob/c3e61a8f4a4a6a094f666e434d994aaa9991bc86/SwiftStyleGuide.swift#L106) 🤦🏼‍♂️